plugins/woocommerce-admin/client/marketplace/components/extensions/extensions.tsx

```diff
@@ -17,60 +18,75 @@ import { Product, ProductType } from '../product-list/types';
 import { MARKETPLACE_ITEMS_PER_PAGE } from '../constants';
 
 interface ExtensionsProps {
+	categorySelector?: boolean;
 	products?: Product[];
 	perPage?: number;
+	label: string;
+	labelPlural: string;
 }
 
 export default function Extensions( props: ExtensionsProps ): JSX.Element {
```

Comment:
Could we rename this to something like `Products`?

Reply:
Done in 6263b0025b807d0d3ea3089b0cb774bad59f61d2